inbm/dispatcher-agent/dispatcher/sota/os_updater.py

```diff
@@ -406,11 +406,11 @@ def update_remote_source(self, uri: Optional[CanonicalUri], signature: Optional[
         return []
 
     def update_local_source(self, file_path: str) -> List[str]:
-        """Concrete class method to create command list to update from a local source for TiberOS OS.
+        """Concrete class method to create command list to update from a local source for TiberOS.
         @param file_path: path to local file
         @return: Command list to execute to perform update.
         """
-        return ["echo FIXME: add actual TiberOS command to update from a local source"]
+        return []
```

Comment:
this should raise an exception: either NotImplementedError or something specific to Sota? (check context to find out)

Reply:
It should be NotImplementedError(). Added.